operator/build.gradle.kts

```diff
@@ -0,0 +1,149 @@
+/*
```

Comment:
Anything that needs to be configured for podman?

Reply:
> Any chance to not let the exception be printed to the console?

The first line comes from here:

https://github.com/quarkiverse/quarkus-operator-sdk/blob/b9f9ccb9a88c31947e169a2ca5d163e5921cdbf8/core/deployment/src/main/java/io/quarkiverse/operatorsdk/deployment/VersionAlignmentCheckingStep.java#L49

The "build time property cannot be changed" message comes probably from some dev service, but I couldn't find which one. See also https://github.com/quarkusio/quarkus/issues/23680.

I don't know how to suppress these messages since they are logged during build, not during tests.

> Got this during the bigtable IT:

Is this happening all the time, or randomly? I've never seen this. I will have to install podman to investigate. It's not related to BigTableIT, but rather to the start of the K3s container before the tests.